canopen/include/co_sync.h

```diff
@@ -38,10 +38,11 @@ extern "C" {
 #define CO_SYNC_COBID_EXT    ((uint32_t)1 << 29)    /*!< extended format     */
```

Comment:
I'm unhappy with `CO_SYNC_COBID_OFF`, because the set indicates: enable generation of SYNC messages; this a relict from previous state I guess, because this is not in the scope of the review ;-)

Can we please change this into: `CO_SYNC_COBID_ON`?